gce-wake.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+# Start all the instances in the specified GCE deployment
+  - hosts: localhost
+    tasks:
+      - name: Deploy gce.py
+        file: src=gce.py dest=./gce.py mode=0744
+      - name: Deploy gce_account.pem
+        file: src=gce_account.pem dest=./gce_account.pem mode=0600
```

Comment:
Not everybody uses the PEM file (df316d612978d1a0009fa0f1ba880b6cabd75a6b) and they probably won't have it in this exact location (mine is `~/projects/paas/creds/gce-account.json`).

Can we parse `secrets.py` (which we know the location of) to get the path from `GCE_PARAMS`?